addons/mail/static/src/models/discuss_view.js

```diff
@@ -100,6 +100,21 @@ registerModel({
             readonly: true,
             required: true,
         }),
+        discussSidebarMailboxHistoryView: one('DiscussSidebarMailboxView', {
+            default: insertAndReplace(),
+            inverse: 'discussViewOwnerAsHistory',
+            isCausal: true,
+        }),
+        discussSidebarMailboxInboxView: one('DiscussSidebarMailboxView', {
+            default: insertAndReplace(),
+            inverse: 'discussViewOwnerAsInbox',
+            isCausal: true,
+        }),
+        discussSidebarMailboxStarredView: one('DiscussSidebarMailboxView', {
+            default: insertAndReplace(),
+            inverse: 'discussViewOwnerAsStarred',
+            isCausal: true,
+        }),
```

Comment:
I wonder if in this case the view suffix shouldn't come bebefore the history/etc suffix?

Reply:
I don't think so. `DiscussSidebarMailboxView` is the traits shared between the view records, but here each field maps to a record. It makes sense that the `view` is at the end, because these records are basically views.

Reply:
Then just name them `inboxView`, `historyView`, `starredView` maybe?

Reply:
Changes done.